### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by luckyboy07** [SuadeLabs/vue3-multiselect#4]
How can i make this library work in typescript?

**Comment by mattelen** [comment]
Great question. I'll look into adding typescript support in the coming weeks

**Comment by luckyboy07** [comment]
Cool! Thanks! great work btw.

**Comment by NinjaTycoon** [comment]
Anyone know a quick and dirty way?  IIRC, we often just need to generate types.  Getting this:
```
TS7016: Could not find a declaration file for module '@suadelabs/vue3-multiselect'. '/home/erik/dev2/svn/ohio-trunk/oc-ui/node_modules/@suadelabs/vue3-multiselect/dist/vue3-multiselect.umd.min.js' implicitly has an 'any' type.
  Try `npm install @types/suadelabs__vue3-multiselect` if it exists or add a new declaration (.d.ts) file containing `declare module '@suadelabs/vue3-multiselect';`
  > 37 |   import Multiselect from '@suadelabs/vue3-multiselect';
```

**Comment by NinjaTycoon** [comment]
Works unless you define :searchable="false".  Then, when you select and item, gives this exception in the console:

```console
runtime-core.esm-bundler.js?5c40:38 [Vue warn]: Unhandled error during execution of watcher callback 
  at <VueMultiselect modelValue=null onUpdate:modelValue=fn track-by="name"  ... > 
  at <PostView> 
  at <ContactUs> 
  at <App>
warn @ runtime-core.esm-bundler.js?5c40:38
logError @ runtime-core.esm-bundler.js?5c40:211
handleError @ runtime-core.esm-bundler.js?5c40:203
callWithErrorHandling @ runtime-core.esm-bundler.js?5c40:157
callWithAsyncErrorHandling @ runtime-core.esm-bundler.js?5c40:163
job @ runtime-core.esm-bundler.js?5c40:2077
flushPreFlushCbs @ runtime-core.esm-bundler.js?5c40:328
flushJobs @ runtime-core.esm-bundler.js?5c40:368
Promise.then (async)
queueFlush @ runtime-core.esm-bundler.js?5c40:286
queueCb @ runtime-core.esm-bundler.js?5c40:308
queuePreFlushCb @ runtime-core.esm-bundler.js?5c40:311
scheduler @ runtime-core.esm-bundler.js?5c40:2105
run @ reactivity.esm-bundler.js?a1e9:183
trigger @ reactivity.esm-bundler.js?a1e9:189
set @ reactivity.esm-bundler.js?a1e9:292
set @ runtime-core.esm-bundler.js?5c40:6219
pointerSet @ vue3-multiselect.umd.min.js?1971:1
eval @ vue3-multiselect.umd.min.js?1971:1
eval @ runtime-dom.esm-bundler.js?830f:1122
callWithErrorHandling @ runtime-core.esm-bundler.js?5c40:154
callWithAsyncErrorHandling @ runtime-core.esm-bundler.js?5c40:163
invoker @ runtime-dom.esm-bundler.js?830f:308
vue3-multiselect.umd.min.js?1971:1 Uncaught (in promise) TypeError: Cannot read property 'setAttribute' of undefined
    at Prox
    y.pointer (vue3-multiselect.umd.min.js?1971:1)
    at callWithErrorHandling (runtime-core.esm-bundler.js?5c40:154)
    at callWithAsyncErrorHandling (runtime-core.esm-bundler.js?5c40:163)
    at Array.job (runtime-core.esm-bundler.js?5c40:2077)
    at flushPreFlushCbs (runtime-core.esm-bundler.js?5c40:328)
    at flushJobs (runtime-core.esm-bundler.js?5c40:368)
```

**Comment by mattelen** [comment]
@luckyboy07 @NinjaTycoon I believe I have fixed this with v1.0.1 - give it a go and let me know how you get on
